apps/fluent-tester/src/FluentTester/theme/applyBrand.ts

```diff
@@ -0,0 +1,102 @@
+import { testerTheme } from './CustomThemes';
+
+export type OfficeBrand = 'Default' | 'Office' | 'Word' | 'Excel' | 'Powerpoint' | 'Outlook';
+type BrandRampKey = 'App1' | 'App2' | 'App3' | 'App4' | 'App5' | 'App6' | 'App7' | 'App8';
+type BrandRamps = { [K in OfficeBrand]: { [J in BrandRampKey]: string } };
+
+const brandColors: BrandRamps = {
```

Comment:
Why should a FluentUI repo know anything about the consumer Office apps? That feels like we're almost reversing the dependency chain by this repo knowing about consts the consumers should define

Reply:
Keep in mind the tester is just test code. This is using the understood Office brands to demonstrate what applying a brand looks like to the components. You can image also adding additional themes / brands in the tester like Material, Spotify, Teams, etc.

Because win32 is exposed and is an Office only tech, some of the fact that we have Office concepts are going to leak into FLURN. As an example we have the shape of the host interface for the win32 palette in @fluentui-react-native/theme-types. Because typedefs have no actual code attached to them though this doesn't incur cost and makes the usage for many office clients easier, even if we don't expect many others to make use of it.